timesketch/api/v1/resources.py

```diff
@@ -410,6 +410,16 @@ def get(self, sketch_id):
             for t in sketch.active_timelines
         ]
 
+        # Get event count and size on disk for each index in the sketch.
+        stats_per_index = {}
+        es_stats = self.datastore.client.indices.stats(
+            index=sketch_indices, metric='docs, store')
+        for index_name, stats in es_stats['indices'].items():
```

Comment:
can we do here

for .... in es_stats.get('indices', {}).items()

just to be more error tolerant.

Reply:
Done.